open_spiel/examples/gtp.cc

```diff
@@ -40,7 +40,7 @@ std::unique_ptr<open_spiel::Bot> MakeBot(
     const open_spiel::Game& game,
     std::shared_ptr<open_spiel::algorithms::Evaluator> evaluator) {
   return std::make_unique<open_spiel::algorithms::MCTSBot>(
-      game, std::move(evaluator), /*uct_c=*/2, /*max_simulations=*/1000,
```

Comment:
This is a strict pessimisation, could you kindly explain a bit why this is necessary?

Reply:
Hi @tkoeppe ,

I think that I brought in this change by accident. During development, there was a build error without this change. But I just checked it again and found that it was unnecessary here.

Thanks for pointing this out! Do I need to make another PR to revert this change, or could you just modify it directly?

Reply:
Let's leave that to @lanctot to decide, thanks!